lib/ecto_ldap/adapter.ex

```diff
@@ -9,6 +10,15 @@ defmodule Ecto.Ldap.Adapter do
     IO.inspect(preprocess)
     IO.inspect(options)
 
+    {:ok, connection} = Exldap.connect
+    {:ok, search_results} = Exldap.search_field(
+        connection,
+        "ou=users,dc=puppetlabs,dc=com",
+        "mail",
+        'jeff.weiss@puppetlabs.com')
```

Comment:
I had a feeling you'd say that!  :-)  I'll address this in a later commit, no worries.  It was a useful library for now, since it gave me quick & dirty access to LDAP for testing purposes.